src/Symfony/Bridge/Twig/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "symfony/security-csrf": "~2.4|~3.0.0",
+        "symfony/security-csrf": "~2.6|~3.0.0",
```

Comment:
the dev requirement for other parts of the security component should also be bumped